### PR TITLE
Products

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**

### CODE REVIEW

**Comment by ajit-sharmaGH** [comment]
image

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge

File: CSS/base/navbar.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.search-bar input {
+    height: 2rem;
+    width: 30vw;
+    outline: none;
+    border: 1px solid var(--dark-primary);
+    border-radius: 5px;
+    padding: 1px 5px;
+    font-family: monospace;
+    font-weight: 600;
+    color: #333;
```
Comment (shaksham08) [inline CSS/base/navbar.css:23]: use color variables

File: pages/login.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <div class="logo">
+            <h1>Lifestyle&nbsp;Store</h1>
+        </div>
+        <div class="search-bar"><input type="text" placeholder="search here" /> </div>
+        <ul class="navlink">
+            <li>
+                <a href="../index.html">Home</a>
+            </li>
+
+            <!-- <li>
```
Comment (shaksham08) [inline pages/login.html:28]: Remove unnecessary comments

File: pages/signup.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            <h1>Lifestyle&nbsp;Store</h1>
+        </div>
+        <div class="search-bar"><input type="text" placeholder="search here" /> </div>
+        <ul class="navlink">
+            <li>
+                <a href="../index.html">Home</a>
+            </li>
+
+            <!-- <li>
+                <a href="cart.html"><i class="fa-solid fa-cart-arrow-down"></i></a>
```
Comment (shaksham08) [inline pages/signup.html:29]: Same remove comments if not needed

File: style.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
+/* @import url("CSS/base/navbar.css"); */
```
Comment (shaksham08) [inline style.css:1]: Here also

**Review by shaksham08 — APPROVED** [review]
Added some changes.

**Pull request merged**